lib/eslint/eslint.js

```diff
@@ -878,7 +885,7 @@ class ESLint {
 
 		/** @type {LintResult[]} */
 		let results;
-		const workerCount = exports.calculateWorkerCount(
+		const workerCount = module.exports.calculateWorkerCount(
```

Comment:
Can't we do this here?
```suggestion
		const workerCount = calculateWorkerCount(
```

Reply:
This doesn't work because `calculateWorkerCount` always refers to the function declared in this file, whereas `module.exports.calculateWorkerCount` can be reassigned for testing.

Reply:
Ah gotcha. Maybe add a comment at this line explaining that?

Reply:
Done (c95b345b570966581d374ff8d737418744dc5c2e).